common/consistenthash/consistenthash.go

```diff
@@ -22,6 +22,17 @@ func (r *Ring) search(key string) int {
     // YOUR CODE GOES HERE //
     /////////////////////////
 
+    r.Lock()
+    defer r.Unlock()
+
+    strHash := hashId(key)
+
+    for i, node := range r.Nodes {
+        if node.HashId > strHash {
+            return i
+        }
+    }
```

Comment:
In this case I thought the returned index would be 0, since the nodes are disposed in a ring, so that the node that comes after the last node is the first one (which has index 0). Is it OK to consider that?